projector-agent/build.gradle.kts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2019-2021, JetBrains s.r.o. and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation. JetBrains designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact JetBrains, Na Hrebenech II 1718/10, Prague, 14000, Czech Republic
+ * if you need additional information or have any questions.
+ */
+
+import java.util.*
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+  kotlin("jvm")
+  `maven-publish`
+}
+
+publishing {
+  publications {
+    create<MavenPublication>("maven") {
+      from(components["java"])
+    }
+  }
+}
+
+kotlin {
+  explicitApi()
+}
+
+val compileKotlin: KotlinCompile by tasks
+val targetJvm: String by project
+compileKotlin.kotlinOptions.jvmTarget = targetJvm
```

Comment:
Let's improve it a bit: move it to a common `build.gradle.kts` like it's done in projector-client already: https://github.com/JetBrains/projector-client/blob/501fab63b5bf15b8def0e0114816933215daf269/build.gradle.kts#L57-L61

Reply:
it was already [there](https://github.com/JetBrains/projector-server/blob/master/build.gradle.kts#L56-L58) =D